lib/Runtime/Library/ForInObjectEnumerator.cpp

```diff
@@ -194,7 +194,7 @@ namespace Js
                     else
                     {
                         ScriptContext* scriptContext = currentIndex->GetScriptContext();
-                        scriptContext->GetOrAddPropertyRecord(currentIndex->GetString(), currentIndex->GetLength(), &propRecord);
+                        scriptContext->GetOrAddPropertyRecord(currentIndex, &propRecord);
```

Comment:
the 'if' already checks for VirtualTable - now that is not needed anymore now, right?

Reply:
@akroshg sorry this one is not clear to me?